src/patternfly/components/Page/page.scss

```diff
@@ -369,6 +369,10 @@
   overflow-x: hidden;
   overflow-y: auto;
   -webkit-overflow-scrolling: touch;
+
+  &:target {
```

Comment:
@seanforyou23 and I are reviewing this, and the question came up about whether we need this or not. After some discussion, we're leaning toward not including `&:target` and just always having `outline: 0;` set for the main region.

Reply:
+1 - realized that the style rule doesn't apply when sending focus to the main content area programmatically with JavaScript, which we need to do after route changes to ensure assistive technology is made aware of the new content. If it's ok, I'd like to remove it. WDYT @mcoker ?

Reply:
Sounds good to me!

Reply:
Ended up going with `:focus`, this most closely represents the state of the element we're looking to style and signals to developers the purpose of the style. Let me know if you see any issues with this.